openpype/plugins/publish/validate_unique_subsets.py

```diff
@@ -26,10 +26,11 @@ class ValidateSubsetUniqueness(pyblish.api.ContextPlugin):
     families = ["*"]
 
     def process(self, context):
-
         # Find instance per (asset,subset)
         instance_per_asset_subset = defaultdict(list)
         for instance in context:
+            if "csv" in instance.data["families"]:
+                continue
```

Comment:
Why - suddenly if something is `csv` do we allow multiple instances publishing to the same asset with the same subset? That seems, actually, invalid?

Reply:
Initially, I thought it was a result of leftover changes from early development and possibly incorrect data collection. However, I soon realized that there could be a scenario where multiple versions of a single subset name could be ingested simultaneously under one asset. This would lead to the occurrence of multiple instances with the same subset and asset.

Reply:
I see, that's neat! :)

Then instead of this "hotfix" we should actually be updating the validate unique subsets behavior to also check for the `version` data - and see if those are also unique or not.

Potentially a warning could still be logged for cases where multiple versions are being ingested to the same subset? At least that'd also clarify what's going on.

Reply:
Good point, I will create separate PR for the change. Thanks!

Reply:
here is the PR https://github.com/ynput/OpenPype/pull/6100. I will remove the `csv` check in this PR.

Reply:
Voilá ab61fbeecfe95f52ba11011645ffa9c12803f8cc